distsql/request_builder.go

```diff
@@ -307,9 +306,7 @@ func (builder *RequestBuilder) SetFromInfoSchema(pis interface{}) *RequestBuilde
 
 // SetResourceGroupTagger sets the request resource group tagger.
 func (builder *RequestBuilder) SetResourceGroupTagger(sc *stmtctx.StatementContext) *RequestBuilder {
```

Comment:
A better API design should be `SetResourceGroupTagger(resourceGroupTagger)`

cc @mornyx

Reply:
LGTM, the declaration of this method can be simplified now.